### PR TITLE
add tests for product price printing javascript task

### PR DESCRIPTION
**Pull request by Morenayd**

### CODE REVIEW

File: tests/conditionals_js/conditionals_test.js [hunk 1]
```diff
@@ -0,0 +1,42 @@
+var file_path = process.env.FILE_PATH
+var webdriver = require('selenium-webdriver');
```
Comment (ToluAkin) [inline tests/conditionals_js/conditionals_test.js:2]: Instead of `var` use `const` @Morenayd

File: tests/product_price_printing/test.js [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+    this.timeout(3 * 1000 * 60)
+    var driver = new webdriver.Builder().forBrowser('chrome').build()
+
+    before(function setupWebdriver(done) {
+        var file_url = `file://${file_path}`;
+        driver.get(file_url).then(done)
+    })
+
+    after(function () {
+        // driver.quit()
```
Comment (ToluAkin) [inline tests/product_price_printing/test.js:17]: I think this should be uncommented

File: tests/product_price_printing/test.js [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+    })
+
+    it("prints product price", async () => {
+        var productList = [{ productName: "a", imageUrl: "b", productPrice: "4" }, { productName: "a", imageUrl: "b", productPrice: "4" }, { productName: "a", imageUrl: "b", productPrice: "4" }]
```
Comment (ToluAkin) [inline tests/product_price_printing/test.js:21]: You are not using `await` in your task. You should remove `async`

Comment (ToluAkin) [inline tests/product_price_printing/test.js:21]: You are not using `await` in your task. You should remove `async`

File: tests/product_price_printing/test.js [hunk 4]
```diff
@@ -0,0 +1,24 @@
+var file_path = process.env.FILE_PATH
+const webdriver = require("selenium-webdriver");
+const assert = require("chai").assert
+const chromedriver = require('chromedriver')
+const { Builder, By } = require("selenium-webdriver");
```
Comment (ToluAkin) [inline tests/product_price_printing/test.js:5]: You are not using `assert`, `chromedriver` and `By` in this test. You should remove it.

**Review by ToluAkin — CHANGES_REQUESTED** [review]

**Comment by Ireti203** [comment]
Weldone @Morenayd 

So, I think you should try to set up the repo on your computer so you can also run the test at your end. You ought to have a bash script in the folder to run the test. You can check previous tests in this repo to see the structure and how it works.

File: tests/conditionals_js/conditionals_test.js [hunk 5]
```diff
@@ -0,0 +1,42 @@
...
+        before(function setupWebdriver(done) {
+            var file_url = `file://${file_path}`;
+            driver.get(file_url).then(done);
+        });
+
+        after(function() {
+            driver.quit();
+        });
+
+    it('test case: Prints out the value of array', async () => {
```
Comment (ToluAkin) [inline tests/conditionals_js/conditionals_test.js:22]: This block of code is not well formatted @Morenayd

**Review by ToluAkin — CHANGES_REQUESTED** [review]
Can you add a screenshot of the result of the test to your PR? Thanks.

File: tests/conditionals_js/test.sh [hunk 6]
```diff
@@ -0,0 +1,43 @@
...
+
+set_folder
+
+# Run Tests
+
+customReportDir=$test_folder
+customReportFilename=logfile
+
+echo -e "${ITALIC}visit${NONE} ${BOLD}${UNDERLINE}$test_folder/logfile.html${NONE} in your browser to see test reports"
+env FILE_PATH=$submission_directory/$folder_name/$html_file mocha $PWD/landing_page.js --reporter mochawesome --reporter-options reportDir=$customReportDir,reportFilename=$customReportFilename,quiet=true
```
Comment (Ireti203) [inline tests/conditionals_js/test.sh:43]: Kindly update this like of code appropriately. The script you want to run is conditionals_test.js and not landing_page.js

File: tests/product_price_printing/test.sh [hunk 7]
```diff
@@ -0,0 +1,43 @@
...
+
+set_folder
+
+# Run Tests
+
+customReportDir=$test_folder
+customReportFilename=logfile
+
+echo -e "${ITALIC}visit${NONE} ${BOLD}${UNDERLINE}$test_folder/logfile.html${NONE} in your browser to see test reports"
+env FILE_PATH=$submission_directory/$folder_name/$html_file mocha $PWD/landing_page.js --reporter mochawesome --reporter-options reportDir=$customReportDir,reportFilename=$customReportFilename,quiet=true
```
Comment (Ireti203) [inline tests/product_price_printing/test.sh:43]: Same here @Morenayd

**Review by Ireti203 — CHANGES_REQUESTED** [review]

**Pull request merged**